<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIWindow</class>
 <widget class="QDialog" name="AIWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 43, 43);</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;AI (O)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn15">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn14">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>20</width>
     <height>431</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn11">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn12">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>31</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Vs&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="AIgameStatus">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Final&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn13">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerturn_1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;It's Player's Turn&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn18">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn17">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="u">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#ffffff;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
